addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -402,14 +402,13 @@ def test_message_get_suggested_recipients(self):
         """ Test default creation values returned for suggested recipient. """
         ticket = self.ticket_record.with_user(self.env.user)
         ticket.message_unsubscribe(ticket.user_id.partner_id.ids)
-        suggestions = ticket._message_get_suggested_recipients()
+        suggestions = ticket._message_get_suggested_recipients(no_create=True)
```

Comment:
change not needed

Reply:
actually as the default value is probably going to change (once everything is finished, default recipients call will automatically create partners in order to use a many2many-like in the UI), I make it explicit now :)